fog/overseer/server/src/worker.rs

```diff
@@ -235,12 +246,23 @@ where
     /// Returns the latest round of ingest summaries for each
     /// FogIngestGrpcClient that communicates with a node that is online.
     fn retrieve_ingest_summary_node_mappings(
-        &self,
+        &mut self,
```

Comment:
Could this potentially lead to situation where this function fails multiple mutable references exist to the parent object?

Reply:
I don't understand. Can you explain more?

Reply:
FWIW the single-mutable-borrow constraint is enforced at compile time.